business_requirement_crm/models/business.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models
+
+
+class CrmLead(models.Model):
+    _inherit = "crm.lead"
+
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='set null',
+    )
+    resource_cost_total = fields.Float(
+        compute='_get_resource_cost_total',
+        string='Total Revenue from BR',
```

Comment:
@xie8899 
this is confusing here. 
I agree this is the Revenue but the methods and names are confusing (you name them "resource").
you should have _get_deliverable_price_total
and resource_cost_total should be deliverable_price_total (please check my comment on PR#138)
in general use resource and cost on one side and deliverable and price on the other side.